packages/app/src/runs/RunsSkeleton.vue

```diff
@@ -1,43 +1,67 @@
 <template>
-  <div
-    data-cy="runs-loader"
+  <ul
+    v-if="isUsingGit"
+    data-cy="runsSkeleton-git"
+    class="flex flex-col mb-[24px] gap-[16px] relative before:content-[''] before:absolute before:top-[20px] before:bottom-[10px] before:w-[2px] before:border-2 before:border-dashed before:border-l-0 before:border-y-0 before:border-r-gray-100 before:left-[7px]"
   >
-    <div
-      v-for="i in numberOfLines"
+    <li
+      v-for="i in numberOfLinesGit"
       :key="i"
-      class="mb-4"
     >
-      <ListRowHeader
-        disabled
-        class="border border-gray-100"
+      <div
+        class="flex items-center my-[10px] gap-[8px] children:h-[8px]"
+        :class="{ 'mt-0': i === 0 }"
       >
-        <template #icon>
-          <div class="h-[20px] w-[20px] bg-gray-100 rounded-xl mx-[2px]" />
-        </template>
-        <template #header>
-          <div class="w-[256px] rounded-xl h-[16px] bg-gray-50 my-[4px]" />
-        </template>
-        <template #description>
-          <div
-            v-for="j in 4"
-            :key="j"
-            class="rounded-lg h-[8px] bg-gray-50 mr-[20px] inline-block"
-            :class="j === 4 ? 'w-[64px]' : 'w-[80px]'"
-          />
-        </template>
-        <template #right>
-          <div
-            v-if="i >= 2"
-            class="rounded-lg h-[16px] bg-gray-50 w-[216px]"
-          />
-        </template>
-      </ListRowHeader>
-    </div>
-  </div>
+        <div
+          class="relative w-[8px] rounded-lg bg-gray-50 ml-[4px]"
```

Comment:
```suggestion
          class="relative w-[16px] rounded-lg bg-gray-50"
```
Increase width of "icon" and remove left margin to make this skeleton better match the icon